assets/js-examples-new/page-objects/google/searchResults.js

```diff
@@ -13,7 +13,7 @@
  */
 
 const util = require('util');
-const menuXpath = '//div[contains(@class, "hdtb-mitem")][contains(., "%s")]';
+const menuXpath = '//*[@role = "navigation"]//span[text()="%s"]';
```

Comment:
```suggestion
// starting xpath with './/' tells browser to begin search from the current element,
// while starting with '//' tells browser to begin search from the start of html document.
const menuXpath = './/span[contains(text(), "%s")]'; 
```

Reply:
Thanks for reviewing my PR.
I have worked on the suggestions mentioned above and pushed the changes would you mind re-reviewing it.